analyses/molecular-subtyping-HGG/README.md

```diff
@@ -0,0 +1,66 @@
+## Molecular subtyping for high-grade glioma (HGG) 
+
+This module is adapted from: [`d3b-center/OpenPedCan-analysis`](https://github.com/d3b-center/OpenPedCan-analysis).
+
+
+#### Running the analysis
+
+The analysis can be run with the following (assuming you are in the analysis directory):
+
+```
+bash run-molecular-subtyping.sh
+```
+
+### Standard of HGG molecular subtype
+
+1. If there was an _H3-3A_ K28M, _H3C2_ K28M, _H3C3_ K28M, _H3C14_ K28M, _H3-3A_ K28I, _H3C2_ K28I, _H3C3_ K28I or _H3C14_ K28I mutation -> `DMG, H3K28`
```

Comment:
```suggestion
1. If there was an _H3-3A_ K28M, _H3C2_ K28M, _H3C3_ K28M, _H3C14_ K28M, _H3-3A_ K28I, _H3C2_ K28I, _H3C3_ K28I or _H3C14_ K28I mutation -> `DMG, H3 K28`
```

Reply:
These comments make sense to me. Thank you for pointing out! I have made changes accordingly.